### ISSUE TITLE
[Feature Request] Proton drive integration

### ISSUE THREAD

**Issue by lazerleif** [zhanghai/MaterialFiles#891]
Is it possible to add integration with Proton drive?

**Comment by jbrianc** [comment]
ProtonDrive already supports this via select & Share

https://twitter.com/ProtonDrive/status/1621590622153973761

**Comment by lazerleif** [comment]
Yes that is a way to add files to proton drive, but would be great to have the content browseable like with nextcloud.

I cant upload folders with this "workaround"

**Comment by russianbot91** [comment]
I would like this as well. I'm able to integrate pCloud.
